Doc/library/sqlite3.rst

```diff
@@ -509,15 +545,30 @@ Connection Objects
 
    .. method:: create_aggregate(name, /, n_arg, aggregate_class)
 
-      Creates a user-defined aggregate function.
+      Create a user-defined SQL aggregate function.
+
+      :param name:
+          The name of the SQL aggregate function.
+      :type name: str
+
+      :param n_arg:
+          The number of parameters the SQL aggregate function can accept.
+          If ``-1``, it may take any number of arguments.
+      :type na_rg: int
 
-      The aggregate class must implement a ``step`` method, which accepts the number
-      of parameters *n_arg* (if *n_arg* is -1, the function may take
-      any number of arguments), and a ``finalize`` method which will return the
-      final result of the aggregate.
+      :param aggregate_class:
+          A class that implements the following methods:
 
-      The ``finalize`` method can return any of
-      :ref:`the types natively supported by SQLite <sqlite3-types>`.
+          * ``step()``: Add a row to the aggregate.
+          * ``finalize()``: Return the final result of the aggregate as
+            :ref:`a type natively supported by SQLite <sqlite3-types>`.
+
+          The ``step()`` method accept *n_arg* number of parameters,
+          unless *n_arg* is ``-1``,
+          in which case it may take any number of arguments.
```

Comment:
```suggestion
          The number of arguments that the ``step()`` method must accept
          is controlled by *n_arg*.
```

Instead of repeating the same information here, I'd suggest just deferring to the description of that parameter. Also, fix grammar error (plural agreement, "method accept") and  parameters -> arguments.